main_f4.c

```diff
@@ -431,6 +550,12 @@ main(void)
 		if (board_test_force_pin())
 			continue;
 
+#if INTERFACE_USART
+		/* if the USART port RX line is still receiving a break, just loop back */
+		if (board_test_usart_receiving_break())
```

Comment:
This makes sense - but it may also need to wait out the break or handle the data of the break in the bootloader code by ignoring the data with a timeout.